kge/job/train.py

```diff
@@ -98,17 +93,36 @@ def __init__(
             for f in Job.job_created_hooks:
                 f(self)
 
+    def _initialize_training(self, config: Config, dataset: Dataset, model=None):
+        from kge.job import EvaluationJob
+
+        if model is None:
+            self.model: KgeModel = KgeModel.create(config, dataset)
+        else:
+            self.model = model
+        self.optimizer = KgeOptimizer.create(config, self.model)
+        self.kge_lr_scheduler = KgeLRScheduler(config, self.optimizer)
+        valid_conf = config.clone()
+        valid_conf.set("job.type", "eval")
+        if self.config.get("valid.split") != "":
+            valid_conf.set("eval.split", self.config.get("valid.split"))
+        valid_conf.set("eval.trace_level", self.config.get("valid.trace_level"))
+        self.valid_job = EvaluationJob.create(
+            valid_conf, dataset, parent_job=self, model=self.model
+        )
+        self.model.train()
```

Comment:
This last line should probably always be done.

Reply:
fixed